packages/bpk-component-calendar/DOCS.md

```diff
@@ -0,0 +1,23 @@
+Default export is `BpkCalendarContainer` - This provides an out of the box calendar.
+
+HOCs - 
+  
+  - `withCalendarState` default export creates calendar using all our exports to construct component - which keeps track of the calendar state. In theory a custom state manager could be used for the calendar but would require a lot more effort to create and maintain
```

Comment:
"create and maintain" To be clear this is for the consumer, we provide `withCalendarState` to make it easier for them

Reply:
Would `from a consumer perspective` be a good addition to add on at the end?

Reply:
Yes 👍

Reply:
All done and added 😄